python/grass/grassdb/history.py

```diff
@@ -59,11 +59,13 @@ def read_history(history_path):
     return hist
 
 
-def update_history(command, history_path=None):
-    """Update history file.
+def add_entry_to_history(command, history_path=None):
+    """Add entry to history file.
 
-    :param command: the command given as a string
+    :param str command: the command given as a string
+    :param str|None history_path: history file path string
     """
+    fileHistory = None
```

Comment:
```suggestion
    file_history = None
```
Let's keep the naming conventions consistent.